canonical/vanilla-framework
# Remove remaining deprecated docs

## Done

Removed docs for deprecated patterns that we'd missed previously

## QA

- See that checks pass
- Search the code base for "deprecated". It should only appear in the following instances:
  - in the "What's new" section as a log of which version it was deprecated in
  - in the Upgrade guide, which will be updated in a separate PR prior to release to say that these patterns have been removed, not just deprecated
  - in relation to the "Deprecated" label

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)
